### PR TITLE
Adding data-wrangler illustration

### LINKED ISSUE #2928: Add Scriberia Illustration to data-wrangler page
Add Scriberia illustration to this [Data Wrangler page](https://the-turing-way.netlify.app/collaboration/research-infrastructure-roles/data-wrangler.html) by following these steps: https://the-turing-way.netlify.app/community-handbook/style/style-figures.html

### PR DESCRIPTION
**Pull request by Rainiefantasy**
Adding data-wrangler jpg Scriberia illustration for the Data Wrangler chapter under Research Infrastructure Roles/Collaboration section.

### Summary

Fixes #2928 <NUM>

*Add Scriberia Illustration to data-wrangler page*

### List of changes proposed in this PR (pull-request)

- [x] Add jpg file to figures folder
- [x]  Add details in the _figure-list.md file in book/website/figures directory
- [x] Review all components of figure (image file location, size, name, alt text and title)
- [x] Add figure to Turing Way chapter using MyST syntax

### What should a reviewer concentrate their feedback on?

- [ ] Check formatting of figure components
- [ ] Everything looks ok?

### Acknowledging contributors

- [x] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/main/README.md#contributors) in the README file.
- [ ] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/main/README.md#contributors) in the README file:

### CODE REVIEW

**Review by RayStick — COMMENTED** [review]
This is looking good - thanks for getting this starting. 
A few things to note:
- _"Add details in the _figure-list.md file in book/website/figures directory"_  - I cannot find this file either, so we should ask the next PR reviewer (someone with write access) if this guidance has changed 
- We are including a DOI that links to the zenodo page but this data-wrangler image is not there yet. Do we need to wait for it to be uploaded before merging this PR?

**Comment by malvikasharan** [comment]
Hi @Rainiefantasy and @RayStick. Thank you for this PR.
> "Add details in the _figure-list.md file in book/website/figures directory" - I cannot find this file either, so we should ask the next PR reviewer (someone with write access) if this guidance has changed
This is a good catch - we removed the file as it was not consistently being maintained - I will remove that text from the Style Guide. Although, I think we need to think about a shared image list with alt text that others can use. I will add that action on an issue - any suggestion is very welcome.
> We are including a DOI that links to the zenodo page but this data-wrangler image is not uploaded yet. Do we need to wait for it to be uploaded before merging this PR?
I will update the file list today. You can meanwhile merge this.
> The file is 1.08MB and it says in [the guidance ](https://the-turing-way.netlify.app/community-handbook/style/style-figures.html)it should be below 1MB, but I do not know how strict that is.
I am not very strict about it, but we can reduce the file size further by using image tools, let me know if you would like me to do that.

**Review by RayStick — APPROVED** [review]

**Pull request merged**
